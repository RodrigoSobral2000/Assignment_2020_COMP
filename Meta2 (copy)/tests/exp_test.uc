int a=5, r=3, eritj2k2=8;

int main(void) {
    char a= -4+3-(-3);
    int b=a+4-(-a);
    //string a="teste";

    //if ( a!=b ) return b;

    //while ( ( a == 3 && b < a ) || a < 3 ) a=b-b-b*2;
    a=b;

    int soma(a, b) { return a+b; }


    return a;
}