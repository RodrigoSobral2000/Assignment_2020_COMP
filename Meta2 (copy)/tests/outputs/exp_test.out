Line 1, col 6: syntax error: =
Line 4, col 11: syntax error: =
Line 5, col 10: syntax error: =
Line 13, col 5: syntax error: int
